network-mux/demo/cardano-ping.hs

```diff
@@ -239,9 +289,14 @@ handshakeDec = do
                       return $ Left $ Refused vn msg
                   _ -> return $ Left $ UnknownTag tag
 
+         32773 -> decodeVersion
+         32774 -> decodeVersion
+         32775 -> decodeVersion
+         32776 -> decodeVersion
```

Comment:
Inlining these numbers is not clear, it's better to do the calculation that leads to them explicitly.

Reply:
I would if I could do any operations on constants in a `case` statement.

Reply:
I think one can do that this way:
```
case a of
  5 -> ...
  6 -> ...
  7 -> ...
  x | y <- f x -> 
```